Combobox.js

```diff
@@ -575,22 +571,50 @@ define([
 			return new ComboPopup({combobox: this});
 		},
 
+		evaluateFilteringConditions: function (inputElement) {
```

Comment:
Oh, also, if that's what the function does, then maybe a better name would be "shouldRunQuery" or "shouldShowDropDown" or something like that?   The current name confused me because the method isn't actually running the filter.